test/hotspot/jtreg/serviceability/sa/sadebugd/ClhsdbTestConnectArgument.java

```diff
@@ -75,7 +75,10 @@ public static void main(String[] args) throws Exception {
             System.out.println(out.getStdout());
             System.err.println(out.getStderr());
 
-            out.stderrShouldBeEmptyIgnoreDeprecatedWarnings();
+            out.shouldMatch("WARNING: --connect is deprecated");
+            // --connect is deprecated.  When removed, can revert to:
+            // out.stderrShouldBeEmptyIgnoreDeprecatedWarnings();
```

Comment:
I don't think this comment is necessary since this test is going way once `--connect` is removed.  Same with this comment in all the other tests.